resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerAspect.java

```diff
@@ -68,124 +60,63 @@
 @Aspect
 public class CircuitBreakerAspect implements EmbeddedValueResolverAware, Ordered {
 
-    private static final Logger logger = LoggerFactory.getLogger(CircuitBreakerAspect.class);
-
     private final CircuitBreakerConfigurationProperties circuitBreakerProperties;
-    private final CircuitBreakerRegistry circuitBreakerRegistry;
-    private final @Nullable
-    List<CircuitBreakerAspectExt> circuitBreakerAspectExtList;
-    private final FallbackDecorators fallbackDecorators;
-    private StringValueResolver embeddedValueResolver;
+    private final CircuitBreakerDecorator circuitBreakerDecorator;
 
     public CircuitBreakerAspect(CircuitBreakerConfigurationProperties circuitBreakerProperties,
         CircuitBreakerRegistry circuitBreakerRegistry,
-        @Autowired(required = false) List<CircuitBreakerAspectExt> circuitBreakerAspectExtList,
+        @Autowired(required = false) List<CircuitBreakerDecoratorExt> circuitBreakerDecoratorExtList,
         FallbackDecorators fallbackDecorators) {
         this.circuitBreakerProperties = circuitBreakerProperties;
-        this.circuitBreakerRegistry = circuitBreakerRegistry;
-        this.circuitBreakerAspectExtList = circuitBreakerAspectExtList;
-        this.fallbackDecorators = fallbackDecorators;
+        this.circuitBreakerDecorator = new
```

Comment:
How about making this object as bean?

Reply:
No need, or? Nobody else needs this class.

Reply:
Oh then no needs.